core/src/avm2.rs

```diff
@@ -628,6 +665,13 @@ impl<'gc> Avm2<'gc> {
 
     #[cfg(not(feature = "avm_debug"))]
     pub const fn set_show_debug_output(&self, _visible: bool) {}
+
+    /// Gets the public namespace, versioned based on the current root SWF.
+    /// See `AvmCore::findPublicNamespace()`
+    /// https://github.com/adobe/avmplus/blob/858d034a3bd3a54d9b70909386435cf4aec81d21/core/AvmCore.cpp#L5809C25-L5809C25
+    pub fn find_public_namespace(&self) -> Namespace<'gc> {
+        self.public_namespaces[&self.root_api_version]
```

Comment:
IIRC avmplus doesn't do this, but just making sure intuitively: what stops us from storing the one versioned namespace directly (next to `root_api_version`) instead of looking them up dynamically?

Reply:
We have multiple versioned namespaces in use (almost all items in playerglobals will have the lowest version, but specifically tagged `[API]` definitions will have different versions.